xrpl/models/transactions/account_set.py

```diff
@@ -103,6 +103,9 @@ class AccountSetAsfFlag(int, Enum):
     ASF_DISABLE_INCOMING_TRUSTLINE = 15
     """Disallow other accounts from creating Trustlines directed at this account."""
 
+    ASF_ALLOW_TRUSTLINE_CLAWBACK = 16
```

Comment:
Don't the flags need to be added to the other enums as well?

Reply:
I don't think so, this is just an account flag. Where else could it be added other than `AccountSetAsfFlag`?

Reply:
I was referring to the other flag stuff in this file - `AccountSetFlag` and `AccountSetFlagInterface`

Reply:
These two interfaces you mentioned are meant to support legacy flags I believe. Account flags that have the prefix `tf` are legacy flags, and are used for the `Flags` param of the `AccountSet` transaction as bitmasks. But since the demand for new account flags is high, the 32 bitmask flag param will run out of usable bits eventually. That's why `AccountSet` uses the `SetFlag` field(not bitmask) for all new flags.